src/org/rascalmpl/library/lang/rascal/syntax/Rascal.rsc

```diff
@@ -571,8 +571,17 @@ syntax StringLiteral
 	| interpolated: PreStringChars pre Expression expression StringTail tail 
 	| nonInterpolated: StringConstant constant ;
 
+lexical MultilineCommentContent
+    =  (![*/] | [*] !>> [/] | [/] !>> [*])
```

Comment:
great! that `[*/]` is fine, but perhaps you could try adding a `+` or `*` as in the old `(![*] | [*] !>> [/])*`, because otherwise the parser will be slow on large comments due to the additional non-determinism). So pls. make the non star and non slash parts "eat" up large portions of the string quickly.

Reply:
Sure, will do.

Reply:
It seems that I am getting some ambiguity issues (an exception), when I add a `+`. 
In particular `Ambiguity(|unknown:///|,"MultilineCommentContent+","pr") ...`

Please note that I still mimic the old version, since it has `MultilineCommentContent*`. The only thing I did was to extract the previous part into the `MultilineCommentContent` terminal and add a new alternative that allowed recursive parsing of comments.

Do you have any suggestions on what to do? 😄

Reply:
It seems that the generated Java file does not change in any case, so I am unsure whether it had an effect.

Reply:
that's not right. it should change. perhaps try again? The ambiguity is expected and might be solved by a follow restriction after the +, like `![*/]+ !>> ![*/]` that way it eagerly eats all the non star, non slash characters in rapid succession.

Reply:
Thanks for the suggestion.
I still unfortunately get the same ambiguity error with `![*/]+ !>> ![*/]` when I compile the parser.

Reply:
Do you think it will help if I use the `>` syntax to disambiguate?

Reply:
In any case, I will try to restructure it so the `*` is closer to the terminals.

Reply:
Certainly the `>` will not help. That only helps for mutually recursive expression-like rules.
If the ambiguity remains, it's an idea to use `analysis::grammars::Ambiguity::diagnose` to find out what the issue is.